### DISCUSSION TITLE
vc-graphics-model载入模型设置lightColor参数会报错

### DISCUSSION THREAD

**Discussion by guojunchao1** [zouyaoji/vue-cesium#401] [Q&A]
vue版本： vite
vue-cesiut版本： 3.1.0

![214ba0a274c2c270d26c6ab82eae675](https://github.com/zouyaoji/vue-cesium/assets/34054903/d2e5d0fd-b69c-483c-80a6-30382caa6545)
![3227b127a214afe04048a2d8ba9765c](https://github.com/zouyaoji/vue-cesium/assets/34054903/8bddf3de-babb-4119-a8e1-c6440d18b697)
![05880885aa61a5a75529f2cc8b9be8e](https://github.com/zouyaoji/vue-cesium/assets/34054903/da56de8e-a746-43ab-afc2-b6e066c580b3)

此外，silhouetteColor参数有时也不能生效
求解决方法

**Comment by zouyaoji** [comment]
color不是传笛卡尔坐标吧
![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/eceb9329-734a-4924-8193-207a13101444)

**Reply by guojunchao1** [reply]
试验了传color的值，同样是报一样的错。
参照官网上确实是传Certesian3的值，并且用原声cesium测试没有问题并且生效。
麻烦可否提供一个vue-cesium上lightColor值如何使用的实例？

**Reply by zouyaoji** [reply]
![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/a66cda6d-c8e9-4b2b-ba4b-1f733d6030fd)

**Comment by guojunchao1** [comment]
感谢您的回复！
目前按照您的写法写上去，还是报同样的错误，如果不加lightColor属性模型就可以正常载入。
另外我用lightColor属性初衷是为了调整模型的亮度，从cesium上传Certesian3值亲测可以；
但是目前从您案例上反馈出来效果是改变模型颜色。
![2305b58cc4e3f3bfe748affa4118fdc](https://github.com/zouyaoji/vue-cesium/assets/34054903/4f55e078-4cf2-406b-824f-20885867573a)

**Reply by zouyaoji** [reply]
那不清楚咋回事了，你用你的模型 然后用原生的cesium试试看看

**Reply by guojunchao1** [reply]
ok  感谢！

**Comment by HHANG52** [comment]
又来冒昧借楼打扰了，请问vc-graphics-model又让模型旋转的方法吗，还是要自己原生的去写啊？
